### PR TITLE
Usage postfix

### PR DESCRIPTION
**Pull request by Reettaphant**
- Fix fetching usage postfix in dev environment

### CODE REVIEW

**Comment by Reettaphant** [comment]
@kenoir This also requires allowing the user to perform iam:GetUser. Should be added to the janus list of things on janus that gives the permissions required to run usages, or to the dev cloudformation template?

File: usage/app/lib/Config.scala [hunk 1]
```diff
@@ -80,7 +80,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
   )
 
   private val iamClient: AmazonIdentityManagement =
-    new AmazonIdentityManagementClient(credentialsProvider)
+    new AmazonIdentityManagementClient(awsCredentials)
```
Comment (kenoir) [inline usage/app/lib/Config.scala]: Why not use `credentialsProvider`?

Comment (Reettaphant) [inline usage/app/lib/Config.scala]: That creates a security token is invalid exception. I assumed it was a problem with using credentials from janus to perform the request.

**Comment by Reettaphant** [comment]
@kenoir I had a chat with Adam and it would be possible to create bespoke grid permissions on janus that would list individually all permissions used to run grid. However, he also pointed out that cloudformation credentials can be used to create dynamo tables the first time usages is run and the user name could be stored in the properties file. Doing it this way would avoid doing the work in janus.

**Comment by kenoir** [comment]
:+1:

**Pull request merged**